samples/highcharts/exporting/apply-stylesheets/demo.js

```diff
@@ -0,0 +1,9 @@
+Highcharts.chart('container', {
+    exporting: {
+        applyStyleSheets: true // Don't loose CSS during exporting
```

Comment:
```suggestion
        applyStyleSheets: true // Don't lose CSS during exporting
```
https://www.merriam-webster.com/grammar/lose-vs-loose-usage